dart/tool/common/generate_common.dart

```diff
@@ -8,11 +8,22 @@ import 'package:markdown/markdown.dart';
 
 import 'src_gen_common.dart';
 
-/// [ApiParseUtil] contains top level parsing utilities
+/// [ApiParseUtil] contains top level parsing utilities.
 class ApiParseUtil {
-  /// Extract the current VM Service version number
-  List<int> parseServiceVersion(List<Node> nodes) {
+  /// Extract the current VM Service version number as a String.
+  static String parseVersionString(List<Node> nodes) {
+    final RegExp regex = new RegExp(r'[\d.]+');
```

Comment:
It'll match the first combo of numbers and periods it encounters. Since we're extracting an h1 line, I feel pretty confident it'll either match the version, or fail catastrophically. The code gen will either fail or the developer will see something odd in the diff.